internal/cmd/schema-registry/command_cluster_upgrade.go

```diff
@@ -39,37 +41,49 @@ func (c *command) newClusterUpgradeCommand() *cobra.Command {
 }
 
 func (c *command) clusterUpgrade(cmd *cobra.Command, _ []string) error {
-	packageDisplayName, err := cmd.Flags().GetString("package")
+	environmentId, err := c.EnvironmentId()
 	if err != nil {
 		return err
 	}
 
-	packageInternalName, err := getPackageInternalName(packageDisplayName)
+	clusterList, _, err := c.V2Client.GetSchemaRegistryClusterByEnvironment(environmentId)
 	if err != nil {
 		return err
 	}
-
-	environmentId, err := c.EnvironmentId()
-	if err != nil {
-		return err
+	if len(clusterList.Data) == 0 {
+		return errors.NewSRNotEnabledError()
 	}
+	cluster := clusterList.GetData()[0]
+	clusterSpec := cluster.GetSpec()
 
-	ctx := context.Background()
-	cluster, err := c.Context.FetchSchemaRegistryByEnvironmentId(ctx, environmentId)
+	packageToUpgradeTo, err := cmd.Flags().GetString("package")
```

Comment:
We try to maintain a 1-1 mapping of flag names to variable names. In the long run, it makes the code a bit more maintainable and easier to read.
```suggestion
	package, err := cmd.Flags().GetString("package")
```